dataland-frontend/src/components/forms/CreateGdvDataset.vue

```diff
@@ -0,0 +1,327 @@
+                  <template v-if="subcategoryVisibility.get(subcategory) ?? true">
+                    <div class="uploadFormSection grid">
+                      <div class="col-3 p-3 topicLabel">
+                        <h4 :id="subcategory.name" class="anchor title">{{ subcategory.label }}</h4>
+                        <div :class="`p-badge badge-${category.color}`">
+                          <span>{{ category.label.toUpperCase() }}</span>
+                        </div>
+                      </div>
+
+                      <div class="col-9 formFields">
+                        <FormKit v-for="field in subcategory.fields" :key="field" type="group" :name="subcategory.name">
+                          <component
+                            v-if="field.showIf(companyAssociatedGdvData.data)"
+                            :is="field.component"
+                            :label="field.label"
+                            :placeholder="field.placeholder"
+                            :description="field.description"
+                            :name="field.name"
+                            :options="field.options"
+                            :required="field.required"
+                            :validation="field.validation"
+                            :validation-label="field.validationLabel"
+                            :reportingPeriod="yearOfDataDate"
+                            :data-test="field.name"
+                            :unit="field.unit"
+                            @field-specific-documents-updated="
+                              updateDocumentList(`${category.name}.${subcategory.name}.${field.name}`, $event)
+                            "
+                            :ref="field.name"
+                          />
+                        </FormKit>
+                      </div>
+                    </div>
+                  </template>
+                </div>
+              </FormKit>
+            </FormKit>
+          </FormKit>
+        </div>
+        <SubmitSideBar>
+          <SubmitButton :formId="formId" />
+          <div v-if="postGdvDataProcessed">
+            <SuccessMessage v-if="uploadSucceded" :messageId="messageCounter" />
+            <FailMessage v-else :message="message" :messageId="messageCounter" />
+          </div>
+
+          <h4 id="topicTitles" class="title pt-3">On this page</h4>
+          <ul>
+            <li v-for="category in gdvDataModel" :key="category">
+              <ul>
+                <li v-for="subcategory in category.subcategories" :key="subcategory">
+                  <a
+                    v-if="subcategoryVisibility.get(subcategory) ?? true"
+                    @click="smoothScroll(`#${subcategory.name}`)"
+                    >{{ subcategory.label }}</a
+                  >
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </SubmitSideBar>
+      </div>
+    </template>
+  </Card>
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/444163f7-62f2-4f97-bd47-85622679d49f)
![image](https://github.com/d-fine/Dataland/assets/104447806/b701367b-299c-48de-96a2-008fc09e37a8)
These empty rolling window components are weird. My feeling it, if there is no data available those should not be shown. What do you think?

Reply:
They should be gone now. 

=> If you upload a gdv dataset via the upload form and leave all these rolling window fields empty, you won't have it on the view page anymore.

Reply:
I did not touch the taxonomie section at all and the link is still there showing an empty table.